setup.py

```diff
@@ -2,21 +2,23 @@
 # -*- coding: utf-8 -*-
 import os
 
+# Get the version from the package
+from flex import VERSION
```

Comment:
This causes problem because this triggers a number of other imports from dependencies which may or may not be installed when processing this file (such as when someone runs `python setup.py install` to install from source).

You could change this to use this pattern from one of my other open source projects: https://github.com/pipermerriam/populus/blob/master/populus/__init__.py#L10

The `version` is declared only in the `setup.py` file and the project uses `pkg_resources` to query the installed version of the library.